.github/workflows/ports_psoc6.yml

```diff
@@ -53,14 +53,33 @@ jobs:
     - uses: actions/checkout@v3
     - name: Download binaries
       uses: actions/download-artifact@v3
-    - name: Setup device
+    - name: Container setup
       run: |
-        cd tools/psoc6
-        cp ../../mpy-psoc6_${{ matrix.board }}_${{ needs.server-build.outputs.commit_sha }}/firmware.hex .
-        source mpy-psoc6.sh && openocd_download_install && openocd_board_conf_download ${{ matrix.board }} && mpy_firmware_deploy ${{ matrix.board }} firmware.hex
-        cd ../..
-    - name: Run tests
-      run: cd tests && ./run-tests.py --target psoc6 --device /dev/ttyACM0 -d psoc6
+        echo ${{ secrets.SELF_HOSTED_PASSWORD }} | sudo -S chmod 666 /var/run/docker.sock
+        source tools/ci.sh && ci_psoc6_setup
+    - name: Setup devices
+      run: |
+        cp mpy-psoc6_${{ matrix.board }}_${{ needs.server-build.outputs.commit_sha }}/firmware.hex .
+        source tools/ci.sh && ci_psoc6_flash_multiple_devices firmware.hex
+    - name: Run psoc6 multi test
```

Comment:
This would need atleast two boards flashed right? And then the tests run on different instances. This is already working or?

Reply:
Yes. This is taking by default `/dev/ttyACM0` and `/dev/ttyACM1`.
We can easily rework to pass them instead as arguments (i.e. when the HIL has more serial devices attached).
https://github.com/Infineon/micropython/pull/76/files#diff-398f66844239bcfec34a0362acfee807d8d697c15490291b03bce4904182a45fR231-R238

Reply:
ok..that was my next question then. Understood!